gensim/models/tfidfmodel.py

```diff
@@ -50,7 +53,8 @@ class TfidfModel(interfaces.TransformationABC):
     """
 
     def __init__(self, corpus=None, id2word=None, dictionary=None,
-                 wlocal=utils.identity, wglobal=df2idf, normalize=True):
+                 wlocal=utils.identity, wglobal=df2idf, normalize=True,
+                 pivot_norm=False, slope=0.65, pivot=None):
         """
         Compute tf-idf by multiplying a local component (term frequency) with a
```

Comment:
Need to update docstring according to new parameters (and in general, convert all docstrings to numpy-style here), useful links https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt, http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html